src/__tests__/global.test.js

```diff
@@ -4,4 +4,13 @@ describe('Global', () => {
   it('should return specific properties', () => {
     expect(global).toMatchSnapshot('specific properties');
   });
+
+  it('should set a one-time mutable OPTIONS object', () => {
+    const { OPTIONS } = global;
+    OPTIONS._set = { lorem: 'ipsum' };
```

Comment:
required: expand the test to highlight it remains mutable until `_set` is called